.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
 - pip install -r requirements/tox.txt
 - pip install tox-travis
 script:
-- tox
+- tox -- -n 2 --durations=10
```

Comment:
Ah thank you for explaining. Nose has a timeout concept, which is what I originally thought this was.